integration-tests/testsetups/ocr.go

```diff
@@ -70,7 +71,7 @@ func NewOCRSoakTest(inputs *OCRSoakTestInputs) *OCRSoakTest {
 }
 
 // Setup sets up the test environment, deploying contracts and funding chainlink nodes
-func (t *OCRSoakTest) Setup(env *environment.Environment) {
+func (t *OCRSoakTest) Setup(env *environment.Environment, forwarderFlow bool) {
```

Comment:
Is it possible that this flag can be part of the env? we can already infer it from ETH_USE_FORWARDER config var

Reply:
Seems like it can also be part of the soaktest struct when initializing the test the first time

Reply:
Since it defaults to false anyway

Reply:
> Is it possible that this flag can be part of the env? we can already infer it from ETH_USE_FORWARDERS config var

I would suggest not connect those two things, we might want to easy verify negative route.

Reply:
Added change I can propose with moving forwarderFlow to soak struct.